services/frontend-service/src/ui/components/SideBar/SideBar.tsx

```diff
@@ -235,17 +237,19 @@ export const SideBar: React.FC<{ className?: string; toggleSidebar: () => void }
             });
             setLockMessage('');
         }
-        api.batchService()
-            .ProcessBatch({ actions })
-            .then((result) => {
-                deleteAllActions();
-                showSnackbarSuccess('Actions were applied successfully');
-            })
-            .catch(() => {
-                showSnackbarError('Actions were not applied. Please try again');
-            });
-        handleClose();
-    }, [actions, api, handleClose, lockCreationList, lockMessage]);
+        if (authReady) {
+            api.batchService()
+                .ProcessBatch({ actions }, authHeader)
```

Comment:
how does this work if we dont use azure auth? is authHeader empty?

Reply:
yes: https://github.com/freiheit-com/kuberpult/blob/main/services/frontend-service/src/ui/utils/AzureAuthProvider.tsx#L113